### PR TITLE
BRICR-dev changes into develop

### PR DESCRIPTION
**Pull request by nllong**
**Do not remove this branch after merging down as it will be actively maintained until BRICR simulations are complete (~end of FY).**

#### Any background context you want to provide?
Merging of scenarios were not working correctly.

#### What's this PR do?
* Fixes merging of scenarios (from buildingsync file)
* Merging of meters from BuildingSync files

#### How should this be manually tested?
TBD

#### What are the relevant tickets?
N/A

### CODE REVIEW

File: seed/models/properties.py [hunk 1]
```diff
@@ -693,12 +695,43 @@ def merge_relationships(cls, merged_state, state1, state2):
...
+                    # first get all readings
+                    readings = MeterReadingClass.objects.filter(meter_id=m.id)
+
+                    # then duplicate
+                    meter = MeterClass.objects.get(pk=m.id)
+                    meter.pk = None
+                    meter.scenario_id = scenario.id
+                    meter.save()  # save to get new id / association
+
+                    new_readings = {
```
Comment (adrian-lara) [inline seed/models/properties.py]: To keep things DRY, I'd suggest using the Meter model's `copy_readings` method here with the `overlap_possible` flag set to `False`.

https://github.com/SEED-platform/seed/blob/develop/seed/models/meters.py#L99

Comment (nllong) [reply seed/models/properties.py]: done.

File: seed/models/properties.py [hunk 2]
```diff
@@ -693,12 +695,43 @@ def merge_relationships(cls, merged_state, state1, state2):
...
+
+                # first get meters and meterreadings
+                meters = MeterClass.objects.filter(scenario_id=scenario_id)
+
                 scenario.pk = None
                 scenario.property_state = merged_state
                 scenario.save()  # save to get new id
 
+                # this feels wasteful but I guess it is the pattern...
+                for m in meters:
```
[the remark this answers is not in the document]
Comment (nllong) [reply seed/models/properties.py]: @adrian-lara -- do you want to handle this one? I cleaned up the meter copying portion.

[the remark this answers is not in the document]
Comment (adrian-lara) [reply seed/models/properties.py]: Nevermind, I found some here: https://github.com/BuildingSync/schema/tree/develop/examples

[the remark this answers is not in the document]
Comment (nllong) [reply seed/models/properties.py]: actually, use these though: https://github.com/SEED-platform/seed/tree/bricr-dev/seed/building_sync/tests/data

[the remark this answers is not in the document]
Comment (adrian-lara) [reply seed/models/properties.py]: I believe this is good to go. I used the "Update with BuildingSync" Action in the inventory detail page to test that meters and meter readings are copied to what look to be new scenarios. It's probably worth your double-check though as I may not be reading the data/results correctly.

[the remark this answers is not in the document]
Comment (nllong) [reply seed/models/properties.py]: thanks, i'll take a look!

**Comment by nllong** [comment]
> In response to the changes I didn't make... 😄

For some reason you were tagged for a review, so I just re-requested you... thanks!

**Pull request merged**